test/Microsoft.ML.CpuMath.UnitTests/UnitTests.cs

```diff
@@ -23,13 +24,13 @@ public class CpuMathUtilsUnitTests : BaseTestClass
         private static readonly FloatEqualityComparer _comparer;
         private static readonly FloatEqualityComparerForMatMul _matMulComparer;
         private static readonly string _defaultMode = "defaultMode";
-#if NETCOREAPP3_1
         private static Dictionary<string, string> _disableAvxEnvironmentVariables;
         private static Dictionary<string, string> _disableAvxAndSseEnvironmentVariables;
         private static readonly string _disableAvx = "COMPlus_EnableAVX";
         private static readonly string _disableSse = "COMPlus_EnableSSE";
         private static readonly string _disableAvxAndSse = "COMPlus_EnableHWIntrinsic";
-#endif
+
+        public static bool SkipAvxSse => RuntimeInformation.ProcessArchitecture == Architecture.Arm || RuntimeInformation.ProcessArchitecture == Architecture.Arm64;
```

Comment:
>SkipAvxSse [](http://example.com/codeflow?start=27&length=10)

Would make sense to call this something like SkipAvxSseOnArm? so anyone readying the usage code will understand when this would be set?

Reply:
I didn't set that because I wasn't sure if that was the only time (in the long run) that it would be set. Currently thats the only time we set it, but like if we have to set that in Blazer and we have named the variable arm I think it would be more confusing. Thoughts?

I think what you are saying does make sense though if we won't have to disable this anywhere else.

Reply:
Also, in .netcore 3.1 and newer we can directly ask if the CPU supports those instructions. We can't do that in .netcore 2.1, so as soon as we drop support for 2.1 we can change this call to directly ask the cpu instead of hardcoding like this. I think that will make the most sense.